boards/nxp/imx93_evk/imx93_evk_mimx9352_a55.dts

```diff
@@ -82,18 +82,40 @@
 };
 
 
-&lpi2c1{
+&lpi2c1 {
 	status = "disabled";
 	clock-frequency = <I2C_BITRATE_FAST>;
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-names = "default";
 };
 
-&lpi2c2{
-	status = "disabled";
+&lpi2c2 {
+	status = "okay";
 	clock-frequency = <I2C_BITRATE_FAST>;
 	pinctrl-0 = <&i2c2_default>;
 	pinctrl-names = "default";
+
+	mfd0:adp5585@34 {
+		compatible = "adi,adp5585";
+		reg = <0x34>;
+
+		gpio_exp0: adp5585_gpio {
+			compatible = "adi,adp5585-gpio";
+			gpio-controller;
+			#gpio-cells = <2>;
+			ngpios = <10>;
```

Comment:
As there are three reserved gpios, so ngpios should be 13.
Add annotation here like:  Onchip GPIO R0~R4 are gpio0~gipo4 and onchip GPIO C0~C4 are gipo8~gpio12